server/service/IndividualServicesService.js

```diff
@@ -222,22 +221,18 @@ exports.disregardApplication = function (body, user, originator, xCorrelator, tr
       /****************************************************************************************
        * Preparing response-value-list for response body
        ****************************************************************************************/      
-      let profileUuid = await profile.getUuidListAsync(profile.profileNameEnum.FILE_PROFILE);
-      for (let profileUuidIndex = 0; profileUuidIndex < profileUuid.length; profileUuidIndex++) {
-        uuid = profileUuid[profileUuidIndex];
-        filePath = await fileProfile.getFilePath(uuid)        
-        applicationData = await prepareApplicationData.readApplicationData(filePath)
-        if(applicationData == undefined){
-          throw new Error("File path does not exist")
-        }
-        checkApplicationExists = await prepareApplicationData.isApplicationExist(applicationData, applicationNameRequestBody, releaseNumberRequestBody)
-        if(checkApplicationExists['is-application-exist']){
-          prepareApplicationData.deleteApplication(applicationData["applications"], checkApplicationExists['application-name'])
-          let applicationDataToJson = {
-            "applications": applicationData["applications"]
-          }
-          prepareApplicationData.addAndUpdateApplicationData(filePath,applicationDataToJson)
+      filePath = await fileProfile.getApplicationDataFileContent()
+      applicationData = await prepareApplicationData.readApplicationData(filePath)
+      if (applicationData == undefined) {
+        throw new Error("Application data does not exist")
+      }
+      checkApplicationExists = await prepareApplicationData.isApplicationExist(applicationData, applicationNameRequestBody, releaseNumberRequestBody)
```

Comment:
The function name "isApplicationExist" gives a meaning that it will return a boolean. Whereas , its actually returing an object with 4 variables. Shall we change the function name to "getApplicationDetails" ? If you have a better proposal , please change it accordingly.  
Also modify the variable name checkApplicationExists to "applicationDetails" (or) to a different name that provides context of the holding information. Many thanks.
(Kindly please apply this change in other places aswell, thanks)

Reply:
done